### DISCUSSION TITLE
REST API "contains"

### DISCUSSION THREAD

**Discussion by d0hnj0e** [supabase/supabase#11347] [Questions]
Hi,

when trying the following request

```
## Get Results (Supabase)
curl "https://xxx.supabase.co/rest/v1/dict?word=cs.%7B%22a%22,%22b%22%7D" \
     -H 'apikey: xxx'
```

to find a word containing "a" or "b" in line with the documentation here [(click)](https://postgrest.org/en/stable/api.html), I receive an error 404 (`operator does not exist: text @> unknown`). Could someone kindly assist?

Thanks and all the best!

**Comment by steve-chavez** [comment]
Hello,

contains `@>` only works for array types. You might be able to user [pattern matching](https://postgrest.org/en/stable/api.html#pattern-matching) for this:

```sql
select 'abc' ~ 'a|b';
```

On JS this can be done with the [filter](https://supabase.com/docs/reference/javascript/filter) method:

```js
.filter('col', 'match', 'a|b')
```
